addons/mail/static/src/emoji_picker/emoji_picker.xml

```diff
@@ -18,7 +18,7 @@
                     </t>
                 </t>
             </div>
-            <div class="p-1 d-flex flex-column">
+            <div class="o-mail-EmojiPicker-searchBar p-1 d-flex flex-column">
```

Comment:
`searchBar` => `search` (no need for bar, and it's easier to read when camelCase is avoidable)